### ISSUE TITLE
fix: remove `UNKNOWN` as a feature flag

### ISSUE THREAD

**Issue by ossdhaval** [JanssenProject/jans#6612]
Currently, in the feature flag list in TUI, it shows `UNKNOWN` as a feature flag. As shown below

![image](https://github.com/JanssenProject/jans/assets/343411/11612a64-3167-4a54-aae8-0856aa0aa394)

This is very confusing for the user. Also, it is a technical constraint that is coming out all the way to the user which is not good. 

We either need to remove the need of having UNKNOWN as a flag in the auth server. Or if it is not possible then remove it from the list before presenting it to the user on TUI or any other interface.

**Comment by yuriyz** [comment]
This is enum value on AS. This is not going to be removed from AS code because it represents case when AS failed to recoginize feature flag values (e.g. newer version applied to older AS version or typo or whatever). 
From UI point of view user will find it confusing. I propose hide it on TUI unless there are better ideas.

**Comment by ossdhaval** [comment]
@pujavs config-api is the next layer, can we remove `UNKNOWN` there? TUI is the user interface. The less data it has to manipulate, the better.

**Comment by pujavs** [comment]
@ossdhaval, I can if it is confirmed requirement all all the clients of `config-api`

**Comment by yuriyz** [comment]
We can have general strategy to separate technical response and UI friendly response. We can have additional parameter `ui=true`. By default config-api can return all data "as is". If `ui=true` parameter is present -> manipulate data, e.g. remove `UNKNOWN` as in this case.

**Comment by yuriyz** [comment]
Right, I vote for 2, so we have full control and provide both technical and user-friendly responses.

**Comment by pujavs** [comment]
Implemented endpoint `/jans-config-api/api/v1/jans-auth-server/config/feature-flags` that will return array feature flag except `UNKNOWN`.
This to be used to populate `feature-flags` dropdown.
**Note:**  The corresponding swagger spec will not have the enum values for feature-flag.
